app/js/Routes.jsx

```diff
@@ -7,25 +7,27 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 import React, { Component } from 'react';
-import { Router, Route, Link, IndexRoute, hashHistory, browserHistory } from 'react-router';
```

Comment:
Why remove the space here, edit codes for only this ticket. This can also be seen in other areas of the PR